### ISSUE TITLE
Docs: addons.md doesn't reflect current state

### ISSUE THREAD

**Issue by philoserf** [kubernetes/minikube#2153]
**Is this a BUG REPORT or FEATURE REQUEST?**: Neither

**Minikube version** : v0.23.0
- **OS**:	macOS 10.13.1 build 17B48
- **VM Driver**: virtualbox
- **ISO version**: v0.23.6

My installation says...

```plain
> minikube addons list
- dashboard: enabled
- kube-dns: enabled
- heapster: disabled
- registry: disabled
- addon-manager: enabled
- default-storageclass: enabled
- coredns: disabled
- ingress: disabled
- registry-creds: disabled
```

The docs at https://github.com/kubernetes/minikube/blob/master/docs/addons.md say...

```plain
$ minikube addons list
- addon-manager: enabled
- dashboard: enabled
- kube-dns: enabled
- heapster: disabled
- registry-creds: disabled
```

I have forked and will look into providing a pull request.

**Comment by r2d4** [comment]
Thanks!  We would gladly accept one.

**Comment by philoserf** [comment]
Why close the issue before we have resolution?

**Comment by gbraad** [comment]
Fix got merged with #2154

**Comment by philoserf** [comment]
Ah beaten to it. Thanks.
